plugins/woocommerce/client/admin/client/customize-store/index.tsx

```diff
@@ -1,617 +1,273 @@
+		'woocommerce'
+	);
 
 	return (
-		<>
-			<div
-				className={ `woocommerce-customize-store__container woocommerce-customize-store__step-${ currentNodeCssLabel }` }
-			>
-				{ CurrentComponent ? (
-					<CurrentComponent
-						parentMachine={ service }
-						sendEvent={ send }
-						context={ state.context }
-						currentState={ state.value }
+		<div className="woocommerce-customize-store__container">
+			<div className="mobile-header">
+				<Button
+					className="mobile-header__back-button"
+					onClick={ () => {
+						window.location.href = getNewPath( {}, '/', {} );
+					} }
+					icon={ chevronIcon }
+					label={ __( 'Back', 'woocommerce' ) }
+				/>
+				<h1 className="mobile-header__title">
+					{ __( 'Customize your store', 'woocommerce' ) }
+				</h1>
+			</div>
+			<div className="woocommerce-customize-store-sidebar">
+				<motion.div
+					className="woocommerce-edit-site-layout__header-container"
+					animate={ 'view' }
+				>
+					<SiteHub
+						variants={ {
+							view: { x: 0 },
+						} }
+						isTransparent={ false }
+						className="woocommerce-edit-site-layout__hub"
 					/>
-				) : (
-					<CYSSpinner />
-				) }
+				</motion.div>
+
+				<VStack
+					className="woocommerce-edit-site-sidebar-navigation-screen__main"
+					spacing={ 0 }
+					justify="flex-start"
+				>
+					<HStack
+						spacing={ 4 }
+						alignment="flex-start"
+						className="woocommerce-edit-site-sidebar-navigation-screen__title-icon"
+					>
+						<SidebarButton
+							onClick={ () => {
+								window.location.href = getNewPath(
+									{},
+									'/',
+									{}
+								);
+							} }
+							icon={ chevronIcon }
+							label={ __( 'Back', 'woocommerce' ) }
+							showTooltip={ false }
+						/>
+
+						<Heading
+							className="woocommerce-edit-site-sidebar-navigation-screen__title"
+							level={ 1 }
+							as="h1"
+						>
+							{ sidebarTitle }
+						</Heading>
+					</HStack>
+
+					<div className="woocommerce-edit-site-sidebar-navigation-screen__content">
+						<p className="woocommerce-edit-site-sidebar-navigation-screen__description">
+							{ sidebarDescription }
+						</p>
+					</div>
+				</VStack>
+			</div>
+
+			<div className="woocommerce-customize-store-main">
+				<div className="woocommerce-customize-store-banner">
+					<div className="woocommerce-customize-store-banner-content">
+						<div className="banner-actions">
+							<h1>{ __( 'Design your own', 'woocommerce' ) }</h1>
+							<p>
+								{ __(
+									'Quickly create a beautiful store using our built-in store designer. Choose your layout, select a style, and much more.',
+									'woocommerce'
+								) }
+							</p>
+							<Button
+								variant="primary"
+								onClick={ handleDesignClick }
+							>
+								{ __( 'Start designing', 'woocommerce' ) }
+							</Button>
```

Comment:
Not a blocker either, but we should pass an `href` to this component so it's rendered as a link instead of a button. Then, we might need an `e.preventDefault()` in `handleDesignClick`. Besides being more semantically correct, we would get some things "for free" like being able to open in a new tab with Ctrl+click.

Reply:
Excellent suggestion! Done in https://github.com/woocommerce/woocommerce/pull/61991/commits/56ed41ce3edbebf4eb99370722de5a68eec53e48